snippets/terraform/certificates/main.tf

```diff
@@ -90,6 +96,8 @@ resource "azurerm_key_vault_certificate" "example" {
       validity_in_months = 12
     }
   }
+
+  depends_on = [ azurerm_role_assignment.current_user ]
```

Comment:
probably not needed as terraform should handle resource dependencies here.

Reply:
I am seeing this error when that is removed.

Error: checking for presence of existing Certificate "eecfc-e2e-tf-tun445" in https://eecfc-e2e-tf-tun445.vault.azure.net/: keyvault.BaseClient#GetCertificate: Failure responding to request: StatusCode=403 -- Original Error: autorest/azure: Service returned an error. Status=403 Code="Forbidden" Message="Caller is not authorized to perform action on resource.\r\nIf role assignments, deny assignments or role definitions were changed recently, please observe propagation time.\r\n....(not found)\r\nVault: eecfc-e2e-tf-tun445;location=westcentralus\r\n" InnerError={"code":"ForbiddenByRbac"}